pkg/test/resourcefixture/testdata/basic/bigtable/v1beta1/bigtableinstance/bigtableinstanceautoscaling/update.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: bigtable.cnrm.cloud.google.com/v1beta1
+kind: BigtableInstance
+metadata:
+  name: btinstance-${uniqueId}
+spec:
+  displayName: BigtableSample
+  instanceType: PRODUCTION
+  cluster:
+  - clusterId: cluster-1-${uniqueId}
+    zone: us-central1-a
+    autoscalingConfig:
+      cpuTarget: 70
+      maxNodes: 20
+      minNodes: 2
+  # - clusterId: cluster-2-${uniqueId}
+  #   zone: us-west1-b
+  #   autoscalingConfig:
+  #     cpuTarget: 70
+  #     maxNodes: 20
+  #     minNodes: 2
```

Comment:
Shall we uncomment these lines? otherwise the create and update would be the same.